src/server/api/routes/bookstore/index.js

```diff
@@ -0,0 +1,55 @@
+const { Router } = require('express');
+const { Client } = require('@notionhq/client');
+
+const { IS_TESTNET, NOTION_API_KEY } = require('../../../config/config');
+
+const { handleRestfulError } = require('../../middleware/error');
+
+const router = Router();
+
+const notion = new Client({ auth: NOTION_API_KEY });
+
+router.get('/bookstore/listing', async (req, res, next) => {
+  try {
+    const response = await notion.databases.query({
+      database_id: '755267a797e146869f74bf13be788ca5',
+      filter_properties: [
+        'ULp%40', // Class ID
+        'gBQK', // Locale
+        'n%7C%5C%5D', // Listing Date
+        'dqKn', // Featured
+        'WQGW', // Highlighted
+      ],
+      filter: {
+        property: 'Testnet',
+        checkbox: {
+          equals: !!IS_TESTNET,
```

Comment:
Would splitting into 2 db be better? not sure

Reply:
Split